oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -126,6 +126,12 @@ impl Node for GrpcServerNode {
         "grpc-server"
     }
 
+    fn get_privilege(&self) -> NodePrivilege {
+        // This node needs to have `top` privilege to be able to declassify data tagged with a
```

Comment:
nit: I think it would be even clearer to say something like "... to declassify data tagged with any arbitrary user identities"